src/webui/www/public/upload.html

```diff
@@ -6,69 +6,61 @@
 	<script type="text/javascript" src="scripts/mootools-1.2-core-yc.js" charset="utf-8"></script>
-<script type="text/javascript">
-
-function stateChangeHandler() {
-	if (this.readyState == this.DONE) {
-		if (this.status == 200) {
-			window.parent.closeWindows();
-		} else {
-			if (this.responseText != "") {
-				alert(this.responseText);
-			} else {
-				alert("QBT_TR(Upload Failed!)QBT_TR");
-			}
-		}
-		$('upload_spinner').style.display = "none";
-	}
-}
-
-function uploadFiles(files) {
-	var xhr = new XMLHttpRequest();
-	if (xhr.upload) {
-		// start upload
-		var formData = new FormData();
-		for (var i = 0, file; file = files[i]; ++i)
-			formData.append(file.name, file);
-		xhr.onreadystatechange = stateChangeHandler;
-		xhr.open("POST", "command/upload");
-		// Bypass cache
-		xhr.setRequestHeader("Cache-Control", "no-cache");
-		xhr.send(formData);
-	}
-}
-
-function fileHandler(e) {
-	e.preventDefault();
-	$('upload_spinner').style.display = "block";
-	// fetch FileList object
-	var files = $('fileselect').files
-	// process all File objects
-	uploadFiles(files);
-}
-
-window.addEvent('load', function() {
-	// is XHR2 available?
-	var xhr = new XMLHttpRequest();
-	if (xhr.upload) {
-		$('uploadForm').addEvent('submit', fileHandler);
-	} else {
-		$('upload_frame').addEvent('load', function() { window.parent.closeWindows(); });
-		$('uploadForm').addEvent('submit', function() { $('upload_spinner').style.display = "block"; });
-	}
-});
-</script>
 </head>
 <body>
 <iframe id="upload_frame" name="upload_frame" class="invisible" src="javascript:false;"></iframe>
-<form action="command/upload" enctype="multipart/form-data" method="post" id="uploadForm" target="upload_frame" style="text-align: center;">
-<div style="margin-top: 25px; display: inline-block; border: 1px solid lightgrey; border-radius: 4px;">
-	<input type="file" id="fileselect" name="fileselect[]" multiple="multiple" />
+<form action="command/upload" enctype="multipart/form-data" method="post" id="uploadForm" style="text-align: center;" target="upload_frame">
+<p>
+	<div style="margin-top: 25px; display: inline-block; border: 1px solid lightgrey; border-radius: 4px;">
+		<input type="file" id="fileselect" name="fileselect[]" multiple="multiple"/>
+	</div>
+</p>
+<fieldset class="settings" style="border: 0; text-align: left;">
+<div class="formRow" style="margin-top: 12px;">
+	<label for="savepath" class="leftLabelLarge">QBT_TR(Save files to location:)QBT_TR</label>
+	<input type="text" id="savepath" name="savepath" style="width: 16em;"/>
+</div>
+<div class="formRow">
+	<label for="label" class="leftLabelLarge">QBT_TR(Label:)QBT_TR</label>
+	<input type="text" id="label" name="label"/ style="width: 16em;"/>
 </div>
-<div id="submitbutton" style="margin-top: 30px;">
+<div id="submitbutton" style="margin-top: 30px; text-align: center;">
 	<button type="submit" style="font-size: 1em;">QBT_TR(Upload Torrents)QBT_TR</button>
```

Comment:
@sledgehammer999 that's the WebUI window to upload a torrent file.  The window title is "Download local torrent" but the button says "Upload Torrents", so it's weird.  I would change the title to Upload Torrents and @ngosang thinks the button should be changed to Download Torrents.